Makefile

```
@@ -1,46 +1,49 @@
+include $(PWD)/.env
+
 build:
-	docker-compose build
+	docker compose build
+
+up:
+	docker compose up
 
 build-web:
-	docker-compose build web
+	docker compose build web
 
 web:
-	docker-compose up web
-bash:
-	docker-compose exec web bash
+	docker compose up web
 
-
-paster:
-	docker-compose exec web paster --plugin=ckan
+bash:
+	docker compose exec web bash
 
 db-init:
-	docker-compose exec web paster --plugin=ckan db init
+	docker compose exec web ckan -c ckan.ini db init
+
 db-upgrade:
-	docker-compose exec web paster --plugin=ckan db upgrade
+	docker compose exec web ckan -c ckan.ini  db upgrade
+
 rebuild-index:
-	docker-compose exec web paster --plugin=ckan search-index rebuild -r
-issues-init:
-	docker-compose exec web paster --plugin=ckanext-issues issues init_db
+	docker compose exec web ckan -c ckan.ini search-index rebuild
 
+issues-init:
+	docker compose exec web ckan -c ckan.ini issues init_db
 
 ckan:
-	docker build --no-cache --build-arg CKAN_VERSION=2.9.0 -t codeforafrica/ckan:latest -t codeforafrica/ckan:2.9.0 contrib/ckan
+	docker buildx build --platform linux/amd64 --no-cache --build-arg CKAN_VERSION=$(CKAN_VERSION) -t codeforafrica/ckan:latest -t codeforafrica/ckan:$(CKAN_VERSION) -f Dockerfile .
```

```suggestion
	docker buildx build --platform linux/amd64 --no-cache --build-arg CKAN_VERSION=$(CKAN_VERSION) --tag codeforafrica/ckan:latest --tag codeforafrica/ckan:$(CKAN_VERSION) .
```

1. I assume Dockerfile is the default
2. If we're using long form for args e.g. --platform, lets use them throughout.